dotnet/src/Extensions/PromptTemplates.Handlebars/Helpers/KernelHelperUtils.cs

```diff
@@ -64,4 +66,25 @@ public static bool TryParseAnyNumber(string? input)
             double.TryParse(input, out _) ||
             decimal.TryParse(input, out _);
     }
+
+    /// <summary>
+    /// Tries to convert a <see cref="JsonNode"/> object to a specific type.
+    /// </summary>
+    public static object? DeserializeJsonNode(JsonNode? jsonContent)
```

Comment:
Feel free to ignore but I have the distinct feeling this exists already somewhere else (I can't find where, though)